lib/cocoapods/target/pod_target.rb

```diff
@@ -131,6 +141,25 @@ def label
       end
     end
 
+    # @return [Array<FileAccessor>] The list of all files tracked.
+    #
+    def all_files
+      files = [
+        file_accessors.map(&:vendored_frameworks),
+        file_accessors.map(&:vendored_libraries),
+        file_accessors.map(&:resource_bundle_files),
+        file_accessors.map(&:license),
+        file_accessors.map(&:prefix_header),
+        file_accessors.map(&:preserve_paths),
+        file_accessors.map(&:readme),
+        file_accessors.map(&:resources),
+        file_accessors.map(&:source_files),
+        file_accessors.map(&:module_map),
+      ]
+
+      files.flatten.compact.map(&:to_s).uniq
+    end
```

Comment:
maybe this could be part of `FileAccessor`?

Reply:
Could be a static function on `FileAccessor`. Makes more sense in for both `PodDirCleaner` and `PodTarget` instead of making it an instance (or even static) func on `PodTarget`